extensions/2.0/Vendor/MPEG_texture_video/README.md

```diff
@@ -0,0 +1,121 @@
+# MPEG_texture_video
+
+## Contributors
+
+* ISO/IEC SC29 WG3 (MPEG Systems) - Scene Description Breakout Group
+* Contacts
+  * Thomas Stockhammer (MPEG-I Scene Description BoG Chair, tsto@qti.qualcomm.com)
+
+## Status
+
+Based on [ISO/IEC FDIS 23090-14](https://www.iso.org/standard/80900.html)
+
+## Dependencies
+
+Written against the glTF 2.0 spec.
+
+## Overview
+
+`MPEG_texture_video` extension provides the possibility to link a `texture` 
+object defined in glTF 2.0 to media and its respective track, listed by 
+an `MPEG_media`. `MPEG_texture_video` extension also provides a reference 
+to the timed `accessor`, i.e. `accessor` with `MPEG_accessor_timed` extension, 
+where the decoded timed texture will be made available.
+
+When the `MPEG_texture_video` extension is not supported, a texture buffer 
+should filled by data described by the standard glTF source object.
+
+## glTF Schema Updates
+
+**`MPEG_texture_video` Properties**
+
+|   |Type|Description|Required|
+|---|---|---|---|
+|**accessor**|`integer`|Provides the accessor's index in accessors array| &#10003; Yes|
+|**width**|`number`|Provides the maximum width of the texture.| &#10003; Yes|
+|**height**|`number`|Provides the maximum height of the texture.| &#10003; Yes|
+|**format**|`string`|Indicates the format of the pixel data for this video texture.|No, default: `"RGB"`|
+|**extensions**|`object`|JSON object with extension-specific objects.|No|
+|**extras**|[`any`](#reference-any)|Application-specific data.|No|
+
+Additional properties are allowed.
+
+* **JSON schema**: [MPEG_texture_video.schema.json](/Extensions/MPEG_texture_video/schema/MPEG_texture_video.schema.json)
+
+### MPEG_texture_video.accessor
+
+Provides a reference to the `accessor`, by specifying the accessor's index in accessors array, that describes the buffer where the decoded timed texture will be made available. The `accessor` shall have the `MPEG_accessor_timed` extension. The type, componentType, and count of the accessor depend on the width, height, and format.
+
+* **Type**: `integer`
+* **Required**:  &#10003; Yes
+* **Minimum**: ` >= 0`
+
+### MPEG_texture_video.width
+
+Provides the maximum width of the texture.
+
+* **Type**: `number`
+* **Required**:  &#10003; Yes
+
+### MPEG_texture_video.height
+
+Provides the maximum height of the texture.
+
+* **Type**: `number`
+* **Required**:  &#10003; Yes
+
+### MPEG_texture_video.format
+
+Indicates the format of the pixel data for this video texture. The semantics of these values are defined in Table 8.3 of OpenGL specification. The number of components shall match the type indicated by the referenced `accessor`. Normalization of the pixel data shall be indicated by the normalized attribute of the accessor.
```

Comment:
There were dozens of published OpenGL specifications over time. A versioned reference is needed here.

Reply:
implemented version 4.6 as it is the one referenced by ISO/IEC 23090-14.